### PR TITLE
chore(docs): Update README.md

### PR DESCRIPTION
**Pull request by ukd1**
**Why is this PR required? What issue does it fix?**:
It makes the README more readable

**What this PR does?**:
It updates the readme to be:
1. a little more consistent - spacing, use of semi-colons, bash example formatting (i.e. $ vs no $ for examples), use of sudo
2. uses bash / yaml formatting for code blocks instead of none, which helps the reader with a little extra syntax highlighting
3. uses [notes / warning](https://github.com/orgs/community/discussions/16925) blocks to help highlight things of note. However, for anyone reviewing this - the notes and warnings somehow don't look super nice / correct in preview, but do work outside of comments in docs. Here is an example: https://gist.github.com/ukd1/84ffd58ba7c2275713aca7d8081b9d5e
4. some minor grammar changes

**Does this PR require any upgrade changes?**:
No, it's just doc / text changes

**If the changes in this PR are manually verified, list down the scenarios covered:**:
I read re-read the changes, and think they make it better.

### CODE REVIEW

**Comment by sinhaashish** [comment]
@ukd1  : can you please sign your commit.

**Review by sinhaashish — APPROVED** [review]
looks good

**Comment by hrudaya21** [comment]
LGTM, Commit Sign is needed.

File: README.md [hunk 1]
```diff
@@ -23,8 +22,7 @@ See [roadmap](https://github.com/orgs/openebs/projects/10), [e2e-wiki](https://g
 
 ### Prerequisites
 
-Before installing ZFS driver please make sure your Kubernetes Cluster
-must meet the following prerequisites:
+Before installing the ZFS-LocalPV driver please make sure your Kubernetes Cluster meets the following prerequisites :
```
Comment (niladrih) [inline README.md]: nit: whitespace character before the ':'
```suggestion
Before installing the ZFS-LocalPV driver please make sure your Kubernetes Cluster meets the following prerequisites:
```

File: README.md [hunk 2]
```diff
@@ -88,9 +83,8 @@ https://github.com/openebs/zfs-localpv/blob/HEAD/docs/faq.md#6-how-to-add-custom
 
 ### Installation
 
-We can install the latest release of OpenEBS ZFS driver by running the following command.
-
-```
+We can install the latest release of OpenEBS ZFS driver by running the following command :
```
Comment (niladrih) [inline README.md]: nit: whitespace character before ':'
```suggestion
We can install the latest release of OpenEBS ZFS driver by running the following command:
```

**Review by niladrih — CHANGES_REQUESTED** [review]
Looks good, requires a DCO sign-off

**Comment by ukd1** [comment]
> Looks good, requires a DCO sign-off

Is that easy to do?

edit; just saw the link in the ci - will read and do it

**Comment by ukd1** [comment]
@niladrih ✅

**Review by niladrih — APPROVED** [review]
Awesome, LGTM.

**Comment by ukd1** [comment]
BTW @niladrih the changes you made actually make the doc inconsistent - before there were things like `something something :` and `something something:` -- I changed (most? all?) to be ` :` - the changes I just accepted (without recalling why I did them) now make it mixed.

I don't really mind if it's ` :` or `:` before an example, but the mix makes my OCD go nuts. Which do you prefer, and I'll update them all

**Comment by niladrih** [comment]
Sure @ukd1. The issue seems to be missing line breaks. With markdown files, ending lines with two spaces at the end breaks the line. When you update the lines, adding line breaks should work.

Found this here: https://github.com/mattcone/markdown-guide/blob/master/_basic-syntax/line-breaks.md

**Pull request merged**